src/lib/Microsoft.Health.Events/Repository/StorageManager.cs

```diff
@@ -13,12 +14,10 @@ public class StorageManager : IRepositoryManager
     {
         private BlobContainerClient _blobContainer;
 
-        public StorageManager(string connectionString, string blobContainerName)
+        public StorageManager(Uri containerUri, BlobContainerClient containerClient)
         {
-            EnsureArg.IsNotNull(connectionString);
-            EnsureArg.IsNotNull(blobContainerName);
-
-            _blobContainer = new BlobContainerClient(connectionString, blobContainerName);
+            EnsureArg.IsNotNull(containerUri);
+            _blobContainer = containerClient;
```

Comment:
>containerClient [](start = 29, length = 15)

Can we do a _blobContainer = EnsureArg.IsNotNull(containerClient, nameof(containerClient));?